addons/skin.estuary/xml/Variables.xml

```diff
@@ -580,6 +583,7 @@
 		<value condition="Player.HasAudio">$INFO[MusicPlayer.Artist]</value>
 	</variable>
 	<variable name="PlayerLabel2">
+		<value condition="VideoPlayer.Content(episodes) + !String.IsEmpty(VideoPlayer.EpisodePart)">$LOCALIZE[20373]$INFO[VideoPlayer.Season,: , / ]$LOCALIZE[20359]$INFO[VideoPlayer.Episode,/]$INFO[VideoPlayer.EpisodePart,: ]</value>
 		<value condition="VideoPlayer.Content(episodes)">$LOCALIZE[20373]$INFO[VideoPlayer.Season,: , / ]$LOCALIZE[20359]$INFO[VideoPlayer.Episode,: ]</value>
```

Comment:
Remove this change to variable `PlayerLabel2` instead I'm going to suggest a change to variable `PlayerLabel3`

Reply:
Sure I only put it there because seemed related. I still can't trigger it do you have a keymap for Ctrl-N?

Reply:
No special keymap just the default one.

If you want to add it to your own key combo it's action `ActivateWindow(playercontrols)`